api/odl.py

```diff
@@ -146,6 +146,36 @@ class ODLAPIConfiguration(ConfigurationGrouping):
         options=ConfigurationOption.from_enum(HashingAlgorithm),
     )
 
+    prioritized_drm_schemes = ConfigurationMetadata(
+        key=FormatPriorities.PRIORITIZED_DRM_SCHEMES_KEY,
+        label=_("Prioritized DRM schemes"),
+        description=_(
+            "A list of DRM schemes that will be prioritized when OPDS links are generated. "
+            "DRM schemes specified earlier in the list will be prioritized over schemes specified later. "
+            f"Example schemes include <tt>{DeliveryMechanism.LCP_DRM}</tt> for LCP, and <tt>{DeliveryMechanism.ADOBE_DRM}</tt> "
+            f"for Adobe DRM. "
+            "An empty list here specifies backwards-compatible behaviour where no schemes are prioritized."
```

Comment:
Should probably use the American spelling of behavior here.

Reply:
Hehe, oops!

I often give away my nationality like this. :laughing: